multiversx_sdk_core/transaction_factories/delegation_factory_test.py

```diff
@@ -2,44 +2,44 @@
 
 from multiversx_sdk_core.address import Address
 from multiversx_sdk_core.constants import DELEGATION_MANAGER_SC_ADDRESS
+from multiversx_sdk_core.interfaces import IValidatorPublicKey
 from multiversx_sdk_core.messages import ArbitraryMessage
 from multiversx_sdk_core.transaction_factories.delegation_factory import \
     DelegationFactory
 from multiversx_sdk_core.transaction_factories.transaction_factory_config import \
     TransactionFactoryConfig
 
 
+class MockValidatorPublicKey(IValidatorPublicKey):
```

Comment:
If `mypy` does not complain when using `bytes` directly, instead of the mock (given the fact that `IValidatorPublicKey` is a "Protocol" class), then we can remove this mock.